frontend/schemaCMS/src/routes/project/dataSourceList/dataSourceList.component.js

```diff
@@ -198,7 +198,7 @@ export class DataSourceList extends PureComponent {
       [T, always(this.renderCreatedInformation([whenCreated, `${firstName} ${lastName}`]))],
     ])({ metaFailed, jobProcessing, metaProcessing, isUploading, fileUploadingError, jobFailed, noDataSourceUploaded });
 
-  renderItem = ({ name, created, createdBy, id, metaData, activeJob, jobsState = {}, fileName }, index) => {
+  renderItem = ({ name, created, createdBy, id, tags, metaData, activeJob, jobsState = {}, fileName }, index) => {
```

Comment:
Can you make sure this is tested? I think we should have a snapshot showing the number of tags

Reply:
done